test/asm/cpudetect.s

```diff
@@ -24,6 +24,10 @@
    taz
 .endif
 
+.ifpdtv
+   sac #00
```

Comment:
A small nitpick:  please make the operand hexadecimal.

Reply:
Done.